exec/jdbc-all/src/test/java/org/apache/drill/jdbc/ITTestShadedJar.java

```diff
@@ -67,6 +70,33 @@ public void run() {
   }
 
   @Test
+  public void testDatabaseVersion() throws Exception {
+
+    // print class path for debugging
+    System.out.println("java.class.path:");
+    System.out.println(System.getProperty("java.class.path"));
```

Comment:
this class needs a serious refactoring, but I'm hesitant of doing it in that patch